test/k8sT/Services.go

```diff
@@ -294,17 +294,21 @@ var _ = Describe("K8sServicesTest", func() {
 			}
 		}
 
-		It("Tests NodePort (kube-proxy)", func() {
-			testNodePort(false)
-		})
-
-		It("Tests NodePort (kube-proxy) with externalTrafficPolicy=Local", func() {
+		testExternalTrafficPolicyLocal := func() {
```

Comment:
Could you extend the test to check whether a request sent to `k8s2` has been dropped? Can be done as a follow-up.

Reply:
Will do in a separate PR!